lib/src/features/chat/presentation/pages/archieve_page.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:flutter/material.dart';
+import '../../data/repositories/chat_repository_impl.dart'; // 导入存储库实现
+
+// 聊天卡片Widget，用于显示每个聊天项
+class ChatCard extends StatelessWidget {
+  final String userName;
+  final String avatarUrl;
+  final String lastMessage;
+  final VoidCallback? onTap;
+
+  const ChatCard({
+    super.key,
+    required this.userName,
+    required this.avatarUrl,
+    required this.lastMessage,
+    this.onTap,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      margin: const EdgeInsets.symmetric(vertical: 8.0),
+      elevation: 2,
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(15.0),
+      ),
+      child: ListTile(
+        leading: CircleAvatar(
+          radius: 30,
+          backgroundImage: NetworkImage(avatarUrl),
+        ),
+        title: Text(
+          userName,
+          style: const TextStyle(
+            fontWeight: FontWeight.bold,
+            fontSize: 18,
+          ),
+        ),
+        subtitle: Text(
+          lastMessage,
+          style: const TextStyle(
+            color: Colors.grey,
+          ),
+        ),
+        onTap: onTap,
+      ),
+    );
+  }
+}
+
```

Comment:
Separate file, this is a reusable widget

Reply:
I’ve moved the ChatCard widget into its own file (chat_card.dart) under widgets/